src/wazuh_modules/syscollector/tests/sysCollectorImp/syscollectorImp_test.cpp

```diff
@@ -9,6 +9,7 @@
  * Foundation.
  */
 #include <cstdio>
+#include "json.hpp"
```

Comment:
It might be redundant, thank you for the comment. It was added in some particular moment of the development when the compilation requested it in some particular OS.

If these include are harmless, I'd leave them.